packages/user-management/src/standard-user-workflow/core/StandardUserWorkflowAsyncTaskService.ts

```diff
@@ -48,28 +47,31 @@ export class StandardUserWorkflowAsyncTaskService extends Service {
    *  }
    * );
    * @access public
-   * @param accountDetails 
-   * @param confirmationRedirection 
+   * @param accountDetails
+   * @param confirmationRedirection
    */
-  signup(accountDetails: AccountCreationDetails, confirmationRedirection?: Redirection): Promise<PendingAccountConfirmation> {
+  signup(
```

Comment:
Are we going back with prettier ??
If the line hasn't changed, why formatting provides old formatting ?

Reply:
Done